src/unit.test.ts

```diff
@@ -79,147 +142,220 @@ test("kubernetes server operations", async () => {
     expect(nodes.nodes).toBeDefined();
     expect(Array.isArray(nodes.nodes)).toBe(true);
 
-    // Delete test pod if it exists
-    console.log("Deleting test pod if exists...");
-    const deletePodResult = await client.request(
+  });
+
+  /**
+   * Test case: Complete pod lifecycle management
+   * Tests the full lifecycle of a pod including:
+   * 1. Cleanup of existing test pods
+   * 2. Creation of new test pod
+   * 3. Monitoring pod until running state
+   * 4. Verification of pod logs
+   * 5. Pod deletion and termination verification
+   * 
+   * Note: Test timeout is set to 90 seconds to accommodate all operations
```

Comment:
```suggestion
   * Note: Test timeout is set to 120 seconds to accommodate all operations
```

Should keep this global in vitest & not per test